src/App.php

```diff
@@ -220,6 +220,13 @@ static function () use ( $slim ) {
 					$page->setLogs( $slim->logs );
 					$page( $project );
 				} )->name( 'SAL' );
+
+				$slim->get( 'atom/(:project)', function ( $project = 'production' ) use ( $slim ) {
```

Comment:
The new linter rules that I snuck in underneath your PR want this to be `static function ( ...`. `composer fix` would automatically update this for you.